attributes/ldap_config.rb

```diff
@@ -1,4 +1,9 @@
 default['grafana']['ldap_verbose_logging'] = false
+default['grafana']['ldap_multi_tree_auth'] = false
+# When the ldap_multi_tree_auth attribute is set to true, it is possible to configure
```

Comment:
Ack to unblock you, but I suspect that we will need to patch format_config to do that when upstreaming this patch.